unity-client/Assets/Scripts/MainScripts/DCL/Controllers/Scene/ParcelScene.cs

```diff
@@ -566,6 +566,11 @@ public BaseComponent EntityComponentCreateOrUpdate(string entityId, CLASS_ID_COM
                 return null;
             }
 
+            if (classId == CLASS_ID_COMPONENT.AUDIO_SOURCE || classId == CLASS_ID_COMPONENT.AUDIO_STREAM)
+            {
+                SceneController.i.boundariesChecker?.AddEntityToBeChecked(entity);
+            }
```

Comment:
Not for this PR, but we should be configuring these aspect of components on some sort of manifest to avoid all the ifs like this. Take into account we already have this anti-pattern in messaging system -- the conditional voiding the lossy messages depending on message type.

Reply:
I think we can use the same interface and simply check if the `Monobehaviour` retrieved (either by the componentFactory or the entity components list is `IOutOfSceneBoundariesHandler`.

I'm gonna give it a call in this PR and if it gets messy I will leave it and create a ticket.